From 1d49b7cd5e835dd2c205cea4efdb91c0b804bac7 Mon Sep 17 00:00:00 2001
From: Jorge Ramirez-Ortiz <jorge.ramirez@oss.qualcomm.com>
Date: Wed, 5 Mar 2025 16:24:17 +0100
Subject: [PATCH 3/4] arm64: dts: qcom: qcs6490-rb3gen2: Update the LPASS audio
  node

Update the lpassaudio node to support the new compatible as the
lpassaudio needs to support the reset functionality on the
QCS6490 RB3Gen2 board and the rest of the Audio functionality would be
provided from the LPASS firmware.

Reviewed-by: Dmitry Baryshkov <dmitry.baryshkov@linaro.org>
Signed-off-by: Taniya Das <quic_tdas@quicinc.c
Upstream-Status: Submitted [https://lore.kernel.org/linux-arm-msm/20250221-lpass_qcm6490_resets-v5-0-6be0c0949a83@quicinc.com/]
---
 arch/arm64/boot/dts/qcom/qcs6490-rb3gen2.dts | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/qcom/qcs6490-rb3gen2.dts b/arch/arm64/boot/dts/qcom/qcs6490-rb3gen2.dts
index 8431269c913c..a87467a22795 100644
--- a/arch/arm64/boot/dts/qcom/qcs6490-rb3gen2.dts
+++ b/arch/arm64/boot/dts/qcom/qcs6490-rb3gen2.dts
@@ -1251,6 +1251,10 @@ usb_hub_reset_state: usb-hub-reset-state {
                output-low;
                bias-disable;
        };
-
 };
 
+
+&lpass_audiocc {
+	compatible = "qcom,qcm6490-lpassaudiocc";
+	/delete-property/ power-domains;
+};
-- 
2.34.1

